src/services/diagramming.js

```diff
@@ -113,6 +113,11 @@ function diagramming() {
             interactive: interactive
         });
 
+        paper.el.setAttribute('tabindex', '0');
+        paper.el.addEventListener('mousedown', function() {
+            this.focus();
+        });
+
```

Comment:
Had to add the tabindex to make the paper focusable. That way the hotkeys will only execute when the paper is in focus, and not when doing something else like filling out threat details.